app/assets/javascripts/components/overview/available_actions.jsx

```diff
@@ -191,6 +192,12 @@ const AvailableActions = createReactClass({
       ));
     }
 
+    if (user.admin) {
```

Comment:
This should only appear in Wiki Education mode (when `Features.wikiEd` is truthy).